bundle/manifests/operator-certification-operator.clusterserviceversion.yaml

```diff
@@ -239,7 +316,7 @@ spec:
                 - --upstream=http://127.0.0.1:8080/
                 - --logtostderr=true
                 - --v=10
-                image: gcr.io/kubebuilder/kube-rbac-proxy:v0.8.0
+                image: registry.redhat.io/openshift4/ose-kube-rbac-proxy@sha256:f000432f07cd187469f0310e3ed9dcf9a5db2be14b8bab9c5293dd1ee8518176
```

Comment:
Should pinned names like this be committed? It seems like this should be a part of the release process, and not committed to the repo.

Reply:
The release process would then have to go back and commit data after the fact. IE

- Publish an image
- Query the registry for the sha
- Update the make file
- Run make bundle 
- Replace tags with sha's

I personally like having a copy of the bundle in the repo and not just in the `certified-operators` repo.

Reply:
@bcrochet can we set this as resolved or do you still prefer to remove it?